updated description of using session keys with Engine (#6474)
thirdweb-dev/js: apps/portal/src/app/engine/features/account-abstraction/page.mdx

Grant & revoke sessions on an account which allow specific wallets to interact with specified contracts for a given period of time with the `/contract/<chain>/<account_address>/account/sessions` endpoints.

## Transact with an account using session keys

Make write calls with a smart account by calling a write endpoint and specify the `x-backend-wallet-address` and `x-account-address` headers.

> The `x-backend-wallet-address` should be set to the engine backend wallet that has been given by the user admin of the smart account signer permission via a [session key](https://portal.thirdweb.com/references/typescript/v5/erc4337/addSessionKey)

Reviewer: this needs some re-wording / better grammar

```suggestion
The `x-backend-wallet-address` should be set to the Engine backend wallet that has been granted signing permission by the smart account's admin through a [session key](https://portal.thirdweb.com/references/typescript/v5/erc4337/addSessionKey)
```
